src/lib/utils.js

```diff
@@ -1,5 +1,5 @@
 import { jsPsych } from 'jspsych-react'
-import requireContext from 'require-context.macro'
+//import requireContext from 'require-context.macro'
```

Comment:
why comment this out? wouldn't leaving it here but with no images to find just  return an empty array?

Reply:
A bug came up when there were no images. I got stuck on this last night... I think I am going to put 2 blank images in the assets folder (instead of the moneybags) and revert all the changes I made surrounding images